packages/framework/request-handler/src/test/requestHandlers.spec.ts

```diff
@@ -93,7 +93,7 @@ describe("RequestParser", () => {
             const requestParser = RequestParser.create({ url: "/objectId/route", headers: { wait: true } });
             const response = await rootDataStoreRequestHandler(requestParser, runtime);
             assert.equal(response?.status, 200);
-            assert.equal(response.value.route, "/route");
+            assert.equal(response?.value.route, "/route");
```

Comment:
curious why this change as part of this PR?

Reply:
I'm not sure exactly why, but for whatever reason the linter gave me trouble for this one line